Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-findOneAndReplace.txt

MongoDB\\Collection::findOneAndReplace()
========================================

.. meta::

>    :description: Discover how to use `findOneAndReplace()` in PHP to find and replace a document in a MongoDB collection with various options and return the modified document.

Reviewer: ```suggestion
   :description: Find and replace a document in a collection with the MongoDB PHP Library, with various options and return the modified document.
```
